uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp

```diff
@@ -512,7 +519,7 @@
                         <portlet:renderURL var="initialViewWithFilterURL" windowState="MAXIMIZED">
                             <portlet:param name="initialFilter" value="${portletCategory}"/>
                         </portlet:renderURL>
-                        <li>- <a href="${initialViewWithFilterURL}">${portletCategory}</a></li>
+                        <ul><li>- <a href="${initialViewWithFilterURL}">${portletCategory}</a></li></ul>
```

Comment:
@dorisgjata, Beware the `<ul>` needs to be outside of the `<c:forEach></c:forEach>`. Thanks for fixing the missing `<ul>` tag and fix this accessibility and coding issue